src/olympia/conf/dev/settings.py

```diff
@@ -47,8 +47,8 @@
 SITE_URL = 'https://' + DOMAIN
 SERVICES_URL = env('SERVICES_URL',
                    default='https://services.addons-dev.allizom.org')
-STATIC_URL = 'https://addons-dev-cdn.allizom.org/static/'
-MEDIA_URL = 'https://addons-dev-cdn.allizom.org/user-media/'
+STATIC_URL = '%s/static/' % CDN_HOST
+MEDIA_URL = '%s/user-media/' % CDN_HOST
```

Comment:
for some reason I really would love to have us use `urlparse.urljoin` to avoid `//` by misconfiguration ("Hmm, had the host the slash in it or not...") but we can also do that in a cleanup ticket later.